docker-compose.yaml

```diff
@@ -1,6 +1,6 @@
 services:
   haproxy:
-    image: haproxy:latest
+    image: haproxy:lts
```

Comment:
Are we sure we don't want to pin a specific version, since this still feels like we're floating? Maybe the recurring periodic task would be to check and see if whatever our pinned version is (e.g. `3.0`) is still even with the Haproxy `lts` tag, and if not open issue upgrade.